src/handlers/getMetrics.js

```diff
@@ -25,8 +25,8 @@ module.exports = async function honeyPrice(message) {
 
   let { uniswapFactory } = result.data
   let honeyFactoryLiquidity = numberWithSpaces(Math.round(uniswapFactory.totalLiquidityUSD))
-  let honeyFactoryVolume = numberWithSpaces(Math.round(uniswapFactory.dailyVolumeUSD))
-  let honeyFactoryFees = numberWithSpaces(Math.round(uniswapFactory.dailyVolumeUSD * 0.003))
+  let honeyFactoryVolume = numberWithSpaces(Math.round(uniswapFactory.totalVolumeUSD))
+  let honeyFactoryFees = numberWithSpaces(Math.round(uniswapFactory.totalVolumeUSD * 0.0003))
```

Comment:
Sorry I got confused, I thought that dailyVolumeUSD was also part of uniswapFactory.
However, I think that my change related to fees (0.0003 to 0.003) was correct.
@crisog

Reply:
> Sorry I got confused, I thought that dailyVolumeUSD was also part of uniswapFactory.
> However, I think that my change related to fees (0.0003 to 0.003) was correct.
> @crisog

Yep, that one was right, let's create a new PR for it.